x-pack/plugins/security_solution/common/endpoint/models/event.ts

```diff
@@ -96,6 +96,21 @@ export function primaryEventCategory(event: ResolverEvent): string | undefined {
   }
 }
 
+/**
+ * @param event The event to get the full ECS category for
+ */
+export function allEventCategories(event: ResolverEvent): string | string[] | undefined {
+  // Returning "Process" as a catch-all here because it seems pretty general
```

Comment:
i think the comment is wrong.

Reply:
https://github.com/elastic/kibana/pull/70097/commits/ae547c5c83f84f4974049f94abfe73db7db62bdd